fhir-persistence-schema/README.md

```diff
@@ -1,16 +1,15 @@
 # IBM FHIR Server - fhir-persistence-schema
 
-Builds and manages the multi-tenant FHIR R4 RDBMS schema for Db2 and PostgreSQL and includes Derby support for testing.
+Builds and manages the multi-tenant FHIR R4/R4B RDBMS schema for PostgreSQL and includes Derby support for testing.
```

Comment:
```suggestion
Builds and manages the FHIR R4/R4B RDBMS schema for PostgreSQL and includes Derby support for testing.
```

Reply:
Fixed. Also changed other instances of IBM to LinuxForHealth.